testsuite/model/src/test/java/org/keycloak/testsuite/model/infinispan/CacheExpirationTest.java

```diff
@@ -54,6 +53,9 @@
     @Test
     public void testCacheExpiration() throws Exception {
 
+        log.debug("Check if previous instances of the class exist on the heap");
+        getNumberOfInstancesOfClass(AuthenticationSessionAuthNoteUpdateEvent.class);
```

Comment:
@ahus1 This is here just for displaying number of instances in memory or does it have any additional purpose?

Reply:
@martin-kanis - yes, this is the intent. No other purpose. Please comment how I should change this code to make this more clear.

Reply:
I see. Maybe put the `getNumberOfInstancesOfClass` directly into log.debug like `log.debugf("Number of previous instances of the class on the heap:  %d", getNumberOfInstancesOfClass(AuthenticationSessionAuthNoteUpdateEvent.class));`?

Reply:
done, thank you!